app/build.gradle

```diff
@@ -57,6 +57,7 @@ dependencies {
     implementation 'com.android.support.constraint:constraint-layout:1.1.3'
     implementation 'com.android.support:design:27.1.1'
     implementation 'com.android.support:recyclerview-v7:27.1.1'
+    implementation 'com.android.support:cardview-v7:27.1.1'
```

Comment:
At some point, we should add variable for 
```groovy
support_version = '27.1.1'
```

Reply:
I was thinking at that but I did not want to take this in my own hands without asking

Reply:
I agree with the sentiment @jhugman.  It might be premature to do that, as these support packages are now effectively being replaced with [AndroidX](https://developer.android.com/jetpack/androidx/).  I think it's ok to hold off _for now_, since these values **MUST** match the `compileSDK` version or `gradlew check` complains loudly.

I have a branch I've been looking into some of these migrations, and will submit an issue to really track it.